### DISCUSSION TITLE
How can we benchmark salaries and contracts for RSEs?  (Session 3 - Main Room)

### DISCUSSION THREAD

**Discussion by rowlandm** [RSEAA/RSEAA.github.io#130] [Discussion topics for #RSEAA24]
In the [RSE-AUNZ monthly online discussion](https://rse-aunz.org/events/) today we talked about "How do you know you are being paid what is market rates for an RSE?"

This led to a discussion of "How can we benchmark salaries and contracts for RSEs?".

**Comment by rowlandm** [comment]
What was also discussed was "How can we show we provide value/impact as RSEs?"

An example of this might be the ["Estimating the value and impact of Nectar Virtual Laboratories"](https://vuir.vu.edu.au/35517/)

**Reply by klapaukh** [reply]
Perhaps we should survey the conference attendees for their basic demographics, job titles, industries, and salaries? And we can report that at the end of the conference. It's not a formal study or anything, but it will at least give people a chance to get an idea of what else is out there.

Apart from that we (as the local RSE community) may want to create an open register of who is doing what with what salaries. There are a bunch of questions about how we might want to do that, but if it may actually be worth doing as on-going monitoring.

**Reply by klapaukh** [reply]
We discussed running a survey with questions about salary, nature of job roles (do you manage staff, do you develop software), and satisfaction so we can get an idea of the real current state.

**Comment by klapaukh** [comment]
It is commonly said that you should move jobs to get more money / better conditions. That seems to hold true in RSE just as much.

**Comment by klapaukh** [comment]
Is there a market for an RSE consulting firm?

**Reply by rowlandm** [reply]
I think there are in specific domains, but not across domains... eg. BioGrid.
